digital_land/cli.py

```diff
@@ -176,6 +178,8 @@ def dataset_dump_flattened_cmd(ctx, input_path, output_path):
 @dataset_resource_dir
 @organisation_path
 @collection_dir
+@performance_dir
```

Comment:
You probably just need the performance directory not both as the operational issue directory is in the performance directory.